roles/st2repos/tasks/redhat.yml

```diff
@@ -0,0 +1,21 @@
+---
+- name: Define RH Version
+  set_fact:
+      rh_ver: "{{ ansible_distribution_version|int|abs }}"
```

Comment:
It was before I discovered `ansible_distribution_major_version`. This was my way of converting `7.3` to `7`.

`7Server` is something that comes up on the server side, not an Ansible thing. So actually, I'm using `ansible_distribution_version`, which equals `7.3`, and converting it to `7` with these filters in order to make sure the right major version was used in the yum config (instead of relying on the yum variable `releasever`, which did result in `7Server`).

Anyways, I just used `ansible_distribution_major_version` instead of these filters in 6984fae. Achieves same result without the filters